website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -0,0 +1,326 @@
+---
+title: "Databricks configurations"
+id: "databricks-configs"
+---
+
+TODO: fix this
+
+<!----
+To-do:
+- use the reference doc structure for this article/split into separate articles
+--->
+
+## Configuring tables
+
+When materializing a model as `table`, you may include several optional configs that are specific to the dbt-databricks plugin, in addition to the standard [model configs](model-configs).
+
+| Option  | Description                                                                                                                        | Required?               | Example                  |
+|---------|------------------------------------------------------------------------------------------------------------------------------------|-------------------------|--------------------------|
+| file_format | The file format to use when creating tables (`parquet`, `delta`, `hudi`, `csv`, `json`, `text`, `jdbc`, `orc`, `hive` or `libsvm`). | Optional | `delta`|
+| location_root  | The created table uses the specified directory to store its data. The table alias is appended to it.                               | Optional                | `/mnt/root`              |
+| partition_by  | Partition the created table by the specified columns. A directory is created for each partition.                                   | Optional                | `date_day`              |
+| clustered_by  | Each partition in the created table will be split into a fixed number of buckets by the specified columns.                         | Optional               | `country_code`              |
+| buckets  | The number of buckets to create while clustering                                                                                   | Required if `clustered_by` is specified                | `8`              |
+
+## Incremental models
+
+<Changelog>
```

Comment:
We've deprecated the usage of the <Changelog> feature. If there is information specific to a version of dbt, we leverage the <versionBlock> feature. If you'd like to call out that a feature is only available in certain versions of a product or a new feature, feel free to do so in the main body of the doc.